design-proposals/emts-scale-provisioning.md

```diff
@@ -0,0 +1,255 @@
+
+### MVP requirements
+
+Following are the MVP requirements for the scale provisioning of EMT-S edge node supporting OXM workflow:
+
+- Provision multiple BareMetal edge nodes without onboarding for the purpose of standalone/singleton use.
+- Provide deploy a service on the local network that can achieve this provisioning at scale.
+- Deploy the provisioning service on the local network that support PXE Boot (BIOS/UEFI with DHCP + TFTP) boot and iPXE with HTTPs.  
+- Have a UX to pre-register BareMetal edge nodes using Serial number or UUID.
+- Provision different OS profiles to different edge nodes selected based on Serial number or UUID.
+- Provision default OS when a device on the LAN boots over PXE and is not pre-registered.
+- Have a UX of collecting provisioning logs and status of edge nodes.
+- Provide the same UX and EN capabilities as if an [EMT-S node is provisioned via USB](https://github.com/open-edge-platform/edge-microvisor-toolkit-standalone-node/tree/main/standalone-node).
+
+> Note: It might be possible for EMF-EIM to support provisioning of the EMT-S nodes. supporting this capability as part of MVP depends on any active customer requirements.
+
+## Solution
+
+The solution consists of three major enhancements (modifications) to EMF:
+1) **Support legacy PXE boot to scale EMT-S provisioning** - the EIM will be extended with a local DHCP/TFTP server that helps initiate OS provisioning via legacy PXE boot.
+   The legacy PXE boot uses the devices' PXE firmware to bootstrap into iPXE, which then continues the EN provisioning process.
+2) **Extend usage of Platform Bundle to be compatible with USB-based EMT-S** - to be consistent with USB-based EMT-S, the EIM will consume Platform Bundle that contains all the scripts and files to install standalone K8s cluster and other customizations.
+3) **Use EIM standalone deployment** - the proposed solution involves deploying a specific EMF profile, referred to as EIM standalone profile. 
```

Comment:
In 3.1 we deploy the whole orchestrator in the OXM flow reducing the resource requirements (TODO: measure resources needed for 100 within 5 minutes, also unlimited resources as good as we can get numbers.) and use roles NOT to show App and CO elements in UI, default MT user, projects and org created automatically via a module during installation, with a PoC of Infra-only profile of EMF with "extreme" simplification but still using the orchestrator as a whole, not part of 3.1. Reason behind decision to keep the whole of EMF is limited resource/performance/time and UX gain compared to the effort we need to invest to maintain. PoC of Daniele will be revisited for a possible 3.2 implementation.

as per the discussion please move this lower in the ADR and advertise as PoC.